espanol/cocinero/cocinero.md

```diff
@@ -0,0 +1,5 @@
+Beatito encontro al cocinero con cara de preocupado.
+
+[El cocinero huyo sin dar explicaciones](huida/trabajo.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.